### PR TITLE
Amino Audit Outline

### PR DESCRIPTION
**Pull request by taariq**
AMINO Legacy REST endpoint audit.

Rendered:
https://github.com/cosmosdevs/stargate/blob/cosmos-amino-audit-api-endpoints/audit.md

To Do

* [ ] Review from the SDK devs
* [ ] Complete audit of transaction construction endpoints
* [ ] Tendermint endpoints like Block

### CODE REVIEW

File: audit.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+### Summary
+
+The Cosmos Stargate Upgrade will provide 3 interfaces for sending and retrieving transactions.
+
+1. Legacy AMINO JSON REST API
+2. GRPC Clients
+3. GRPC Gateway JSON REST.
+
+The Legacy Amino interface provides an interface most similar to cosmoshub-2 and prior versions of the Cosmos SDK. This interface is available for both querying and sending transactions. It is implemented by executing Amino JSON serialization refliection over the new set of protobuf compatible structs.
```
Comment (ebuchman) [inline audit.md:12]:
```suggestion
The Legacy Amino interface provides an interface most similar to cosmoshub-3 and prior versions of the Cosmos SDK. This interface is available for both querying and sending transactions. It is implemented by executing Amino JSON serialization refliection over the new set of protobuf compatible structs.
```

File: audit.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+The core finding of this audit is that while changes to the underlying structs result in an interface that is close to the prior version.
```
Comment (ebuchman) [inline audit.md:14]: Unfinished sentence?

File: audit.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+      "tokens": "0",
+      "unbonding_time": "1970-01-01T00:00:00Z"
+
+  }
+```   
+#### Delegators
+* **Endpoint Name:** QueryDelegatorDelegations
+* **Endpoint Path:** ```"/staking/delegators/delegations"```
+* **What Changed:**
+  * ```“balance”``` now is no longer a number. It is a field with two values: ```"amount"``` and ```"Denom”```
```
Comment (ebuchman) [inline audit.md:89]: Should `Denom` be capitalized here? I believe JSON fields are all supposed to be lower case but there's been some inconsistency in the past

Comment (taariq) [reply audit.md:89]: That uppercase may have been a bug. I'll check on it.

**Pull request merged**